bittensor/subtensor.py

```diff
@@ -1431,7 +1426,8 @@ def get_existential_deposit(self, block: Optional[int] = None) -> Optional[Balan
         balances below this threshold can be reaped to conserve network resources.
 
         Args:
-            block (Optional[int], optional): Block number at which to query the deposit amount. If ``None``, the current block is used.
+            block (Optional[int], optional): Block number at which to query the deposit amount. If ``None``, the
```

Comment:
Do we need `, optional` if we already have `Optional[int]`?

Reply:
good point. thanks